README.md

```diff
@@ -164,4 +180,5 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+This is a type 2 grammar since it has a list of terminals and non terminals after each state and not just a terminal or 
```

Comment:
But you've shown that it can be parsed with an FSM, which makes it type 3

# -3